ccm_web/server/src/config.ts

```diff
@@ -62,6 +67,26 @@ const prepNumber = (value: string | undefined): string | number | undefined => {
   return (value === undefined) ? undefined : value.trim() === '' ? value : Number(value)
 }
 
+const prepObjectFromJSON = (value: string | undefined): Record<string, unknown > | undefined => {
+  if (value === undefined) {
+    return undefined
+  } else {
+    return JSON.parse(value)
+  }
+}
```

Comment:
Sorry, I missed when you shared the suggestion. But I was listening and did as much I understood. 

The code in the `config.ts` is a learning process for me. The way you tightened up checking on type of values with props. I never gave a deep though on this and always went with an assumption that configuration will be added correct way.

Reply:
I didn't realize at the time that the error would be not very readable, just that it would be caught by the `catch` in `validateConfig`. So you did what I suggested originally!